### ISSUE TITLE
Provide Sustainability Input to QBRs

### ISSUE THREAD

**Issue by jonashuber** [tadodotcom/Sustainability-Team#14]
# Description
- Next to Sales, CX, Voice-of-Customer.... there should be a Sustainability category used as input for prioritising product development

# Related links

# Any other notes?

**Comment by jonashuber** [comment]
2020-04: @MarielaSassi asked Daniel

**Comment by MarielaSassi** [comment]
First asked on April 16th. No answer until today, so I sent a chat message.
If no answer comes in one week, I assume he didn't talk to Dominik and then we can do it now.

**Comment by MarielaSassi** [comment]
Answer from Daniel:
"Had a chat with Dominik on this quite some time ago (mid 2020 If I recall correctly). The item is less about "QBR input" but more about considering environmental aspects (be it intended positive impacts of the avoidance of (unintended) negative impacts) in product development and design decisions. My understanding is that they do apply that lense (among other perspectives) in their decision making, however not sure/aware whether/that there is a structured process around this in place within PM. So feel free to follow up with him."

**Comment by MarielaSassi** [comment]
From the follow-up with Dominik:
"I have two thoughts on this:
a) our products are by themselves already a major sustainability contribution, saving significant amounts of energy and carbon. We always strive to further improve this since it is a key part of our value proposition. The additional leverage by using other materials etc. is most likely negligible.
b) It is already extremely hard for us to move fast and become profitable - any additional overhead adds complexity and costs and slows us down
Bottom line is that at this point in time I don't see free capacity to add this to our process but by "doing what we do" we're already making continued steps towards making heating and cooling much more sustainable."

And we discussed in the sustainability chat to not pursue this at the moment, as he has a completely valid point of now not being the moment to afford the overhead.

Should we close it or put it in backlog? Jonas suggested leaving it for next year...
